kits/ai-pr-reviewer/agent.md

```
@@ -0,0 +1,2 @@
+# AI PR Code Reviewer
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)